bridge/test/utils.js

```diff
@@ -63,6 +65,29 @@ function checkRcpt(tx) {
     checkGas(tx.receipt.gasUsed);
   }
 
+// @param tx is an object returned from the smartcontract tx {
+//  tx: @string,
+//  receipt: {
+//    gasUsed: @number
+//  }
+// }
+// @return BN the gas used multiplied by the gas price
+const getGasUsed = async (tx) => {
+  const gasUsed = new BN(tx.receipt.gasUsed);
+
+  // Obtain gasPrice from the transaction
```

Comment:
I think the code is pretty clear and we could remove the comment (maybe I'm wrong idk)

Reply:
removed ;)